countries.go

```diff
@@ -4894,6 +4894,515 @@ func (c CountryCode) Region() RegionCode { //nolint:gocyclo
 	return RegionUnknown
 }
 
+// Language - return language code of country
+func (c CountryCode) Language() LanguageCode {
+	switch c {
+	case AUS:
+		return LanguageEN
+	case AUT:
+		return LanguageDE
+	case AZE:
+		return LanguageAZ
+	case ALB:
+		return LanguageSQ
+	case DZA:
+		return LanguageAR
+	case ASM:
+		return LanguageSM
+	case AIA:
+		return LanguageEN
+	case AGO:
+		return LanguagePT
+	case AND:
+		return LanguageCA
+	case ATA:
+		return LanguageEN
+	case ATG:
+		return LanguageEN
+	case ARE:
+		return LanguageAR
+	case ARG:
+		return LanguageES
+	case ARM:
+		return LanguageHY
+	case ABW:
+		return LanguageNL
+	case AFG:
+		return LanguageFA
+	case BHS:
+		return LanguageEN
+	case BGD:
+		return LanguageBN
+	case BRB:
+		return LanguageEN
+	case BHR:
+		return LanguageAR
+	case BLR:
+		return LanguageBE
+	case BLZ:
+		return LanguageEN
+	case BEL:
+		return LanguageNL
```

Comment:
I'm sorry, but there is a problem with them method signature

Some countries have multiple official languages

Here's a list of countries with multiple official languages:
 * Africa: South Africa, Algeria, Burundi, Cameroon, Cape Verde, Central African Republic, Comoros, Djibouti, Eritrea, Equatorial Guinea, Kenya, Lesotho, Madagascar, Morocco, Mauritius, Mauritania, Namibia, Uganda, Rwanda, Seychelles, Swaziland, Tanzania, Chad
 * Americas: Canada, Bolivia
 * Asia: India, Singapore, Fiji
 * Europe: Belgium, Switzerland

Your method should return a list of languages

Reply:
that's fair, i can make these adjustments